scripts/automated-translations.ps1

```diff
@@ -168,5 +188,9 @@ else
     Write-Output "`n::group::No pull request`n"
     Write-Output "`n➡︎ No changes to merge to $LtsBranchName branch from $TranslationBranchName branch.`n"
     Write-Output "`n::endgroup::`n"
+    if ($env:GITHUB_OUTPUT)
+    {
+      Write-Output "success=true" >> $env:GITHUB_OUTPUT
+    }
```

Comment:
Adding these `success` outputs so that soft failures could also be caught for unexpected cases.